test/nonnegative_bridge.jl

```diff
@@ -44,7 +45,7 @@ function MOI.supports_constraint(::Type{NonnegativeBridge{T}},
                                  ::Type{Nonnegative}) where T
     return true
 end
-function MOIBC.added_constraint_types(::Type{NonnegativeBridge{T, F}}) where {T, F}
+function MOIB.added_constraint_types(::Type{NonnegativeBridge{T, F}}) where {T, F}
```

Comment:
Seems to work, but I didn't check, whether that's correct.

Reply:
It's correct